src/seedu/addressbook/common/Utils.java

```diff
@@ -34,4 +36,17 @@ public static boolean elementsAreUnique(Collection<?> items) {
         }
         return true;
     }
+
+    /**
+     * Returns a concatenated version of the printable strings of each object.
+     */
+    String getPrintableString(Printable... printables){
```

Comment:
Ideally the getPrintableString(Printable … printable) method should be used somewhere.